### DISCUSSION TITLE
Crashes sometimes

### DISCUSSION THREAD

**Discussion by bokluci** [nxvvvv/safe-exam-browser-bypass#12] [General]
Hello, a month ago it worked fine even during the exam, but now it crashes half the times when I try to start the app. Could that be because I'm still using the old version? Also, could it not allow me to take the exam because it's not the latest version?

**Comment by lathianamc** [comment]
It could be for numerous reasons, without an error message it's impossible to know why it crashes. It could be because your current setup does not match the SEB server policies (e.g. minimum SEB version).
